GitUI/Shells/ShellDescriptor.cs

```diff
@@ -7,13 +7,13 @@ namespace GitUI.Shells
     public abstract class ShellDescriptor : IShellDescriptor
     {
         /// <inheritdoc/>
-        public string? ExecutableCommandLine { get; protected set; } = null!;
+        public string? ExecutableCommandLine { get; protected set; }
```

Comment:
These assignments are redundant. This pattern is commonly used for _non_-nullable fields, where assignment is guaranteed in code that the compiler cannot see (i.e. set via reflection, in the case of DI frameworks such as MEF). In this case the properties are nullable, so the assignment is misleading and redundant.